packages/react-components/src/components/ErrorBar.tsx

```diff
@@ -248,3 +296,38 @@ const errorsToShow = (
     return error.timestamp > dismissal.dismissedAt;
   });
 };
+
+const messageBarType = (errorType: ErrorType): MessageBarType => {
+  switch (errorType) {
+    case 'callNetworkQualityLow':
+    case 'callNoSpeakerFound':
+    case 'callNoMicrophoneFound':
+    case 'callMicrophoneAccessDenied':
```

Comment:
I remember we have this error when we don't have the access to microphone, is this one different? Will there be duplicate warnings?

Reply:
From PR desc - is this being handled in a follow up PR?

Reply:
I checked that there aren't duplicate warnings after this PR.

If the microphone / camera access is denied right at the beginning, the permission banner shows up, but the UFD doesn't fire (because we fail to enumerate the devices entirely)

If the microphone / camera access is denied once the app has loaded, the UFD fires, but the permission banner doesn't show up.

I'll have a follow up PR to merge the permission banner into the `ErrorBar` component, but the two conditions and different (though their error messages should likely be made to be the same).